src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -805,4 +809,56 @@ export class PatientViewPageStore {
         this.ajaxErrors = [];
     }
 
+    readonly trialMatches = remoteData<ITrialMatch[]>({
+        invoke: () => {
+            return postMatchMinerTrialMatches({mrn: this.patientId});
+        },
+        onError: (err: Error) => {
+            // fail silently
+        }
+    }, []);
+
+    readonly trialIds = remoteData<ITrialQuery>({
+        await: () => [
+            this.trialMatches
+        ],
+        invoke: async() => {
+            let nctIds = new Set<string>();
+            let protocolNos = new Set<string>();
+            _.forEach(this.trialMatches.result, (trialMatch: ITrialMatch) => {
```

Comment:
maybe have a comment that explains this.  Might be clearer if I knew what "nct" was

Reply:
```
let nctIds = new Set<string>(); // Trial unique id from clinicaltrials.gov
let protocolNos = new Set<string>(); // trials's MSK ID
```